configure.ac

```diff
@@ -278,13 +278,21 @@ GD_LIB_PKG_CHECK([LIBTIFF], [TIFF], [tiff], [libtiff-4], [
 ])
 
 dnl Check for webp support.
-GD_LIB_CHECK([LIBWEBP], [WEBP], [webp], [
+GD_LIB_PKG_CHECK([LIBWEBP], [WEBP], [webp], [libwebp], [
```

Comment:
looks like you should rebase onto the latest state as this has been merged already

`git fetch && git rebase`